_src/lib/node_cache.coffee

```diff
@@ -644,7 +644,7 @@ module.exports = class NodeCache extends EventEmitter
 				return clone( value.v )
 			else
 				return value.v
-		return null
+		return if value.v == undefined then value.v else null
```

Comment:
This can be written more explicitly:

```coffeescript
_unwrap: ( value, asClone = true )->
  if not @options.useClones
    asClone = false
  if asClone and value.v?
    return clone( value.v )
  else
    return value.v
```